src/watcher/store.ts

```diff
@@ -3,19 +3,23 @@ import { LevelUp } from "levelup";
 import encodingDown from "encoding-down";
 import { LockManager } from "../utils/lock";
 import { Appointment } from "../dataEntities/appointment";
+import EncodingDown from "encoding-down";
```

Comment:
Missed this before: double import for encoding-down.

Reply:
Lol, ty